### DISCUSSION TITLE
Unique frequency tables for each turbine location?

### DISCUSSION THREAD

**Discussion by moptis** [NatLabRockies/floris#480] [Ideas]
I notice that FLORIS now can incorporate heterogenous wind speed conditions, but can it handle heterogenous wind direction conditions? I.e., can each turbine have a unique wind speed / wind direction frequency table? This would be especially useful for assessing AEP offshore, where wind farms are very large and wind direction distributions can vary meaningfully based on distance to shoreline.

Thanks!

Mike

**Comment by kikocorreoso** [comment]
I agree. This could be very useful for sites near-shore, close to islands, sites where measurements have been done far from the site or onshore, etc.

**Comment by paulf81** [comment]
Thank you for this suggestion @moptis , would you mind to add it as a feature-request issue?

**Comment by rafmudaf** [comment]
@moptis this sounds like support for [WRG files](https://windnavigator.ul-renewables.com/products/wrg) as inputs for atmospheric conditions. Is this what you had in mind or something else?
Sorry to bring this up again. I've left it open because it's a good idea and we're currently going through a planning phase for the upcoming year, so your input would be helpful.

**Reply by moptis** [reply]
@rafmudaf Yes WRG support would be awesome and would allow FLORIS to be used in standard pre-construction energy estimate methods.

**Reply by paulf81** [reply]
@bayc @rafmudaf @christiannvaughn  I think this is on the roadmap now right?
